### PR TITLE
set ingredients and steps in functions

### PR DESCRIPTION
**Pull request by ebosetalee**
@jirevwe
couldn't add you as reviewer

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -17,58 +18,36 @@ def selfie():
...
 @app.route("/recipies/<name>", methods=["GET"])
 def get_recipie(name):
-    with open("files/{}.json".format(name), "r") as food:
-        return jsonify({"status": "success", "data": json.load(food)})
+    cippie_db = sqlite3.connect("cippe.db")
+    cursor = cippie_db.cursor()
+    recipies = cursor.execute(
+        "SELECT food_name, description, top_image, bottom_image FROM recipies WHERE food_name=?",
+        (name, ))
+    return jsonify({"status": "success", "data": recipies.fetchall()[0]})
```
Comment (jirevwe) [inline main.py]:
```suggestion
        "SELECT food_name, description, top_image, bottom_image FROM recipies WHERE food_name=? ORDER BY food_name ASC LIMIT 1", (name, ))
    return jsonify({"status": "success", "data": recipies.fetchall()})
```
You can get the first one like this

**Comment by ebosetalee** [comment]
@jirevwe before the next task, anything else?

**Review by jirevwe — APPROVED** [review]

**Comment by jirevwe** [comment]
Nah

**Comment by jirevwe** [comment]
You can merge this, I don't have write access

**Pull request merged**